app/uk/gov/hmrc/servicedependencies/controller/model/Slug.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.servicedependencies.controller.model
+
+import play.api.libs.json.{Json, Reads}
+import uk.gov.hmrc.servicedependencies.model.SlugDependency
+
+case class Slug(uri              : String,
+                name             : String,
+                version          : String,
+                teams            : List[String],
+                runnerVersion    : String,
+                classpath        : String,
+                jdkVersion       : String,
+                dependencies     : List[SlugDependency],
+                applicationConfig: String,
+                slugConfig       : String,
+                latest           : Boolean
+               )
+
+object Slug {
+  implicit val sdReads: Reads[SlugDependency] = Json.reads[SlugDependency]
```

Comment:
I'd be careful with these implicits. There is no reason why importing Slug, should make an implicit Reads[SlugDependency] in scope - it should be scoped as part of the Reads[Slug] implementation.
My preference would to not even make the Reads[Slug] implicit, and make clients choose when to make it implicitly available, since there is no single implementation - it's really defined as part of an Api - not a model fact.
Also, might be beneficial to not use the macro, but explicitly define the Reads, that way it will be obvious if refactoring Slug that external Apis or external mongo references may have been broken.